README.md

```diff
@@ -1,8 +1,33 @@
 # d2 mod for OpenRA
 
-Notes:
-  * The commands given below are meant to be run in a command prompt / terminal from inside of the `OpenRA.Mods.D2` directory.
-
+How to install:
+  * Download OpenRA-Source - Release-20161019  (https://github.com/OpenRA/OpenRA/archive/release-20161019.zip)
+  * Download D2-Source (https://github.com/OpenRA/d2/archive/master.zip)
+  * Extract both files , rename the 'd2-master' folder to 'd2' and move it into the just extracted OpenRA-Source folder  
+  -> OpenRA-release-20161019/mods
+  * Assuming that you already installed OpenRA before from the Website (not from source) , you need to switch to its Directory and copy 3 files : 
+  ```
+    'OpenRA.Game.exe'
+    'Eluant.dll'
+    'OpenRA.Mods.Common.dll' (OpenRA/mods/common)
+  ```
+  * Paste the files into ' /OpenRA-release-20161019/mods/d2/OpenRA.Mods.D2/dependencies'
```

Comment:
If you're updating this, could you also remove the space in front of the first `/` here as well, please?